steve/util.py

```diff
@@ -206,7 +371,17 @@ def get_project_config():
     :returns: config file
 
     """
-    return SteveConfig().load()
+    try:
+        return SteveConfig().load()
+    except ConfigNotFound as e:
+        try:
+            for cfg in [IniConfig, YamlConfig]:
+                if cfg == SteveConfig:
+                    continue  # already tried
+                return cfg().load()
+        except ConfigNotFound:
+            pass
+    raise e   # original raised error
```

Comment:
The situation where 'e' is reraised is when there is no config file at all in the current or parent directory. I think the correct way is then to show that the default config file was not found, instead of the last one tried.